### PR TITLE
Modified LED function of BLE MESH on/off server for C3 and S3 DevKit (IDFGH-7596)

### PR DESCRIPTION
**Pull request by mmt-10**

### CODE REVIEW

**Comment by Alvin1Zhang** [comment]
Thanks for your contribution.

File: examples/bluetooth/esp_ble_mesh/ble_mesh_node/onoff_server/main/board.c [hunk 1]
```diff
@@ -48,6 +84,9 @@ static void board_led_init(void)
         gpio_set_level(led_state[i].pin, LED_OFF);
```
Comment (JiaLWang) [inline examples/bluetooth/esp_ble_mesh/ble_mesh_node/onoff_server/main/board.c:84]: When the chip is C3 or S3, this code segment can not be executed.

File: examples/bluetooth/esp_ble_mesh/ble_mesh_node/onoff_server/main/board.c [hunk 2]
```diff
@@ -21,6 +22,39 @@ struct _led_state led_state[3] = {
...
+        /* Set all LED off to clear all pixels */
+        pStrip_a->clear(pStrip_a, 50);
+    }
+}
+
+void configure_led(void)
+{
+    ESP_LOGI(TAG, "Example configured to blink addressable LED!");
+    /* LED strip initialization with the GPIO and pixels number*/
+    pStrip_a = led_strip_init(0, 8, 1);
```
[the remark this answers is not in the document]
Comment (tatsutaigu) [reply examples/bluetooth/esp_ble_mesh/ble_mesh_node/onoff_server/main/board.c]: Thank you for your review.
I revised it.

File: examples/bluetooth/esp_ble_mesh/ble_mesh_node/onoff_server/main/board.c (hunk 1 above)
Comment (tatsutaigu) [reply examples/bluetooth/esp_ble_mesh/ble_mesh_node/onoff_server/main/board.c:93]: I revised function about board_led_operation().
LED on/off operation is same as ESP32.

**Comment by tatsutaigu** [comment]
This LED issue is not only onoff server but also other ble-mesh examples.
I cannot revise it to all examples because I do not have enough time.

**Comment by JiaLWang** [comment]
> This LED issue is not only onoff server but also other ble-mesh examples. I cannot revise it to all examples because I do not have enough time.

Thanks for your contribution. For other ble-mesh examples, we will update after this PR is merged.

**Pull request closed without merging**